test/fixtures.yml

```diff
@@ -1080,4 +1080,4 @@ worldpay_us:
   merchantpin: "1234567890"
 
 go_cardless:
-  access_token: 'sandbox_mYMHPiOhsiBae0usK-_UToceOjJvpeh-B_zHKKw4'
+  access_token: 'sandbox__BrQwUL1nju54GHmieIjoAjouiGpEWUudwMT63pQ'
```

Comment:
does this change a lot?   expire etc?

Reply:
This will change if you do OAuth authorization in GoCardless, for example when you connect another site in Chargify to same account in GoCardless, they will expire previous token and generate another one.